fix(chat): fixed error message colors

ilaydadastan opened the pull request:
the fix is related to https://github.com/jitsi/jitsi-meet/issues/14573 


after fix the error message looks better

<img width="313" alt="Screenshot 2024-03-29 at 17 21 19" src="https://github.com/jitsi/jitsi-meet/assets/43909097/31c4fc54-dae0-44fa-8c57-7a232e685523">

damencho commented:
@saghul WDYT? 

I think the red color is correctly notifying there was an error, while with this color this is not obvious.

saghul commented:
Yeah I'd like the background to be red-ish as before please. The text can be white.

ilaydadastan commented:
I changed the background as red-ish. I used the red color from  theme.palette because of this color used for other errors like [here](https://github.com/jitsi/jitsi-meet/blob/master/react/features/prejoin/components/web/Prejoin.tsx#L173) 

I also updated the screenshot in the pull request to show how it looks like

ilaydadastan commented:
Hi @saghul @damencho, will we be able to merge it if it's ok in this way?

damencho commented:
Nope. Why did you change the user message?

damencho commented:
I got confused from the first screen shot with the purple.
